msal/src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -197,13 +198,14 @@ public async Task<IAccount> GetAccountAsync(string accountId)
         {
             var accounts = await GetAccountsAsync().ConfigureAwait(false);
             return accounts.FirstOrDefault(account => account.HomeAccountId.Identifier.Equals(accountId, StringComparison.OrdinalIgnoreCase));
-        }
-
+        }
+
         /// <summary>
         /// Attempts to acquire an access token for the <paramref name="account"/> from the user token cache. 
         /// </summary> 
         /// <param name="scopes">Scopes requested to access a protected API</param>
-        /// <param name="account">Account for which the token is requested. <see cref="IAccount"/></param>
+        /// <param name="account">Account for which the token is requested. <see cref="IAccount"/></param>
+        /// <param name="cancellationToken"></param>
```

Comment:
Should provide some explanation. Also pls. provide a link to the wiki which talks about how to use the cancellation Token. Also pls. ensure that the usage of Code Flow including the cancelation token is in a sample. Until release you can validate with internal build and publish the sample when the release is out.

Reply:
aside: Perhaps we should wait adding this everywhere until we are ready to implement it everywhere and then in this change only include changes related to Device Code now?

---
In reply to: [220018243](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1263#discussion_r220018243) [](ancestors = 220018243)